topics/imaging/tutorials/imaging-introduction/tutorial.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: tutorial_hands_on
+
+title: "Introduction to image analysis using Galaxy"
+zenodo_link: https://zenodo.org/record/3358874#.XUQ9oi1XZUM
+questions:
+  - "How do I use Galaxy with imaging data?"
+  - "How do I convert images using Galaxy?"
+  - "How do I display images in Galaxy?"
+  - "How do I filter images in Galaxy?"
+  - "How do I segment simple images in Galaxy?"
+objectives:
+  - "How to handle images in Galaxy."
+  - "How to perform basic image processing in Galaxy."
+requirements:
+time_estimation: "30M"
+
+contributors:
+  - thomaswollmann
+
+---
+
+# Introduction
+{:.no_toc}
+
+This tutorial shows how to use Galaxy for image analysis. The data used in this tutorial is available at [Zenodo](https://zenodo.org/record/3358874#.XUQ9oi1XZUM).
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Getting data
+
+The dataset required for this tutorial contains a screen of DAPI stained HeLa nuclei ([more information](https://zenodo.org/record/3358874#.XUQ9oi1XZUM)). We will use a sample image from this dataset for training basic imaging skills in Galaxy.
```

Comment:
Could you give a bit more of an introduction/motivation? what are these images? what is the research/biological question we are trying to answer in this tutorial?

Reply:
This tutorial is just introductory to image handling there is no research/biological question.

Reply:
it doesn't have to be much, even if it is just "we have an image of cells and want to detect the nuclei and segment the image"

just a thought, is it possible to add a step at the end that counts the number of cells? because that would be a neat little result maybe (but no worries if not possible)